src/order-history/__snapshots__/OrderHistoryPage.test.jsx.snap

```diff
@@ -39,7 +39,7 @@ exports[`<OrderHistoryPage /> Renders correctly in various states renders orders
             <span
               aria-hidden={true}
               className="fa fa-chevron-left mr-2"
-              id="pagination-3"
+              id="pagination-2"
```

Comment:
Do these ids matter and why did they change?

Reply:
They only need to be unique. It seems like maybe our id generator is now 0 indexed somewhere along the way?